lib/pnr.js

```diff
@@ -0,0 +1,43 @@
+class Pnr {
+  static get regex() {
+    return /\b(19|20)?[0-9]{2}[- ]?((0[0-9])|(10|11|12))[- ]?(([0-2][0-9])|(3[0-1])|(([7-8][0-9])|(6[1-9])|(9[0-1])))[- ]?[0-9]{4}\b/g;
```

Comment:
`\b(19|20)?[0-9]{2}((0[0-9])|(10|11|12))(([0-2][0-9])|(3[0-1])|(([7-8][0-9])|(6[1-9])|(9[0-1])))[- ]*[0-9]{4}`

Reply:
Haha, hela målet är att bli av med personnummer i koden och du lägger in det i första kommentaren 😀

Reply:
🤦‍♂️